usdoj-crt/beta-ada
# Expand button

Rationale:

1.  For pages with a large number of accordions (either actual accordion components or details elements) we would like a way for users to easily open and close all accordions.

Changes:

1.  Webpack config has been updated to handle multiple entry points
2.  The details plugin in `/_plugins/details.rb` now sets a data attribute initially. The data attribute is observed by the JS.
3.  New Javascript in `/_assets/js/utils/expand-accordions.js`
4.  New `expand-accordions.html` file in the `_includes` folder. 

Test:

1.  Make sure the expand button doesn't expand any accordions outside of the main content area.
2.  Confirm behavior on mobile (iOS, Android)
3.  Confirm behavior across browsers (Chrome, Firefox, Edge, Safari)
4.  This component needs a11y review from Terri, so do not merge until it has been thoroughly reviewed and tested.
